TauntAndBattleresList.lua

```diff
@@ -9,3 +9,12 @@ addon.tauntList = {
 	[115546] = true, -- Monk: Provoke
 	[185245] = true  -- Demon Hunter: Torment
 }
+
+--@version-retail
+addon.battleresList = {
+	[20484]	 = true, -- Rebirth
+	[20707]  = true, -- Soulstone
+	[61999]	 = true, -- Raise Ally
+	[391054] = true, -- Intercession
```

Comment:
Should add Engineering Combat res to this list instead of keeping it separate.

Reply:
Engi Combat Rezz is tricky bc. it can fail. Therefore I left it out.
What we can do instead and maybe this is even a better solution: 
There should be an event for player resurrection - we can watch for this and check if we are in combat. So - we would not need any special spell treatment, neither any version specific code

Reply:
That sounds like a potentially good solution since all that matters is that someone gets resurrected, especially in retail, as there is no personal CD, just a global CD.

Reply:
Unfortunatly, SPELL_RESURRECT is only present as CLEU on the player beeing resurrected side.
But we can check for aura application of 20484, which is the dummy spell - but we have to check if this method is usable for WotLK & Classic